src/EnergyPlus/HeatingCoils.cc

```diff
@@ -836,7 +837,7 @@ namespace HeatingCoils {
             auto &heatingCoil = state.dataHeatingCoils->HeatingCoil(CoilNum);
             auto &heatingCoilNumericFields = state.dataHeatingCoils->HeatingCoilNumericFields(CoilNum);
             CurrentModuleObject = "Coil:Heating:Gas:MultiStage";
-            heatingCoil.FuelType_Num = Constant::ResourceType::Natural_Gas;
+            heatingCoil.ResourceType = Constant::eResource::NaturalGas;
```

Comment:
Shouldn't this be a `FuelType` and not `ResourceType`? What non-fuel resources are valid here?

Reply:
@dareumnam I feel like this PR is ready to, and needs to, get merged in to avoid conflicts with other refactors.  If this is still a point to be addressed, please add it to the next round.